api/HTMLMediaElement.json

```
@@ -1356,22 +1356,22 @@
               "version_added": "9"
             },
             "opera": {
-              "version_added": true
+              "version_added": "11.6"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": "12"
             },
             "safari": {
-              "version_added": null
+              "version_added": "≤4"
```

The "error" event was in the initial media element support in WebKit:
https://trac.webkit.org/browser/webkit/trunk/WebCore/html/HTMLMediaElement.cpp?rev=27277#L337

So:

```suggestion
              "version_added": "1.3"
```